### PR TITLE
[story/ECP-1236] Dockerize local production environment

### PR DESCRIPTION
**Pull request by nomasprime**

### CODE REVIEW

File: docker-compose.production.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+version: "3.7"
+
+x-env_files:
```
Comment (WorkSutton) [inline docker-compose.production.yml:3]: @nomasprime why `x-env_files`? What does tis represent?

File: .env.production [hunk 2]
```diff
@@ -0,0 +1,18 @@
+DFE_SIGN_IN_ISSUER=https://pp-oidc.signin.education.gov.uk:443
+DFE_SIGN_IN_REDIRECT_BASE_URL=https://localhost:3000
+DFE_SIGN_IN_IDENTIFIER=teacherpayments
+
+DFE_SIGN_IN_API_CLIENT_ID=teacherpayments
+DFE_SIGN_IN_API_ENDPOINT=https://pp-api.signin.education.gov.uk
+
+DQT_CLIENT_HOST=https://s118d01funcappfapp.azurewebsites.net/
+
+ENVIRONMENT_NAME=review
```
Comment (WorkSutton) [inline .env.production:10]: @nomasprime Why is the environment called `review`?

**Comment by WorkSutton** [comment]
@nomasprime @and-lucas-kelly how do we ensure that this is not run in true PRODUCTION?

File: .env.production (hunk 2 above)
Comment (nomasprime) [reply .env.production:10]: Seeds.

File: docker-compose.production.yml (hunk 1 above)
Comment (nomasprime) [reply docker-compose.production.yml:3]: [Extension fields](https://docs.docker.com/compose/compose-file/compose-file-v3/#extension-fields).

**Comment by nomasprime** [comment]
> @nomasprime @and-lucas-kelly how do we ensure that this is not run in true PRODUCTION?
> As I believe it is not meant to be. That rather this is just setup so we can debug locally.

We don't run dotenv in production.

**Review by WorkSutton — APPROVED** [review]

**Pull request merged**